src/scripts/components/surah/AyahsList.js

```diff
@@ -33,6 +39,7 @@ class AyahsList extends React.Component {
   }
 
   render() {
+<<<<<<< HEAD
```

Comment:
Expected an identifier and instead saw '<<'.<br>Expected an operator and instead saw '<<'.<br>Expected an operator and instead saw '<'.<br>Expected an assignment or function call and instead saw an expression.<br>Missing semicolon.